packages/storybook/src/stories/ino-switch/ino-switch.stories.ts

```diff
@@ -38,27 +46,49 @@ export const Playground: Story<Components.InoSwitch> = (args) => html`
   >
     Switch Label
   </ino-switch>
-`;
-Playground.args = {
-  checked: false,
-  disabled: false,
-  name: '',
-};
+`);
+export const Playground = template.generatePlaygroundStory();
 withColorScheme(Playground, 'colorScheme', 'primary');
 
-export const ColorScheme = () => html`
+
+const templateColorSchema = new TemplateGenerator<Components.InoSwitch>(
+  'ino-switch',
+  args => html`
   <ino-switch color-scheme="primary" checked>Primary</ino-switch>
   <ino-switch color-scheme="secondary" checked>Secondary</ino-switch>
   <ino-switch color-scheme="success" checked>Success</ino-switch>
   <ino-switch color-scheme="warning" checked>Warning</ino-switch>
   <ino-switch color-scheme="error" checked>Error</ino-switch>
   <ino-switch color-scheme="light" checked>Light</ino-switch>
   <ino-switch color-scheme="dark" checked>Dark</ino-switch>
-`;
+`);
+/**
+ * Change the color scheme by setting `color-schema` with the following values:
+ * 
+ * - `primary`
+ * - `secondary`
+ * - `success`
+ * - `warning`
+ * - `error`
+ * - `light`
+ * - `dark`
+ */
+ export const ColorScheme = templateColorSchema.generatePlaygroundStory();
 
-export const States = () => html`
+
+const templateStates = new TemplateGenerator<Components.InoSwitch>(
+  'ino-switch',
+  args => html`
   <ino-switch checked>Checked</ino-switch>
   <ino-switch>Unchecked</ino-switch>
   <ino-switch disabled checked>Checked and Disabled</ino-switch>
   <ino-switch disabled>Disabled</ino-switch>
-`;
+`);
+/**
+ * Change the state of the `<ino-switch>` element by setting the following properties
+ * 
+ * - `checked`: Marks this element as checked. (**unmanaged**)
+ * - `disabled`: Disables this element.
+ */
+export const States = templateStates.generatePlaygroundStory();
```

Comment:
Instead of showcasing all states, IMO it might be enough to only show the `disabled` state like this:
```
export const Disabled = template.generateStoryForProp('disabled', true);
```